scheduling/job_scheduling.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+'''
+Author : Mohit Kumar
+Job Sequencing Problem implemented in python
+'''
+
+
+class Scheduling:
+
+    def __init__(self, jobs):
+        self.jobs = jobs
+
+    def schedule(self, n, deadline):
+        self.sdl = deadline
+        self.J = []
+        self.J.append(self.jobs[1])
+        self.x = 2
+        while self.x < n:
+            self.K = self.J.copy()
+            self.K.append(self.jobs[self.x])
+            self.x = self.x + 1
+            self.feasibility = self.feasible(self.K, self.sdl)
+            if self.feasibility is True:
+                self.J = self.K.copy()
+
+        return self.J
+
+    def feasible(self, K, fdl):
+        self.tmp = K
+        self.isFeasible = True
+
+        self.i = 0
+        self.j = 1
+        self.k = 0
+
+        while self.i < len(self.tmp):
+            while self.j < len(self.tmp):
+                self.index1 = self.jobs.index(self.tmp[self.i])
+                self.index2 = self.jobs.index(self.tmp[self.j])
+                self.j = self.j + 1
+                if fdl[self.index1] > fdl[self.index2]:
+                    (self.tmp[self.i], self.tmp[self.j]) = \
```

Comment:
Please format this code with psf/black to remove the backslash.

Reply:
where can i find psf/black formatter  ??